ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -1850,13 +1850,13 @@ private void LoadFont(OpenXmlElement fontSource, IXLFontBase fontBase)
             if (row.CustomFormat != null)
             {
                 Int32 styleIndex = row.StyleIndex != null ? Int32.Parse(row.StyleIndex.InnerText) : -1;
-                if (styleIndex > 0)
+                if (styleIndex >= 0)
```

Comment:
Why `>=`? As far as I know, `0` is reserved for the default style.

Reply:
There were problems with many ClosedXML.Report tests because of this. Style 0 is the default style of a workbook but it does not necessarily equal to `XLStyle.Default` and it is stored in the file just like any other style.

See the example:
![image](https://user-images.githubusercontent.com/19576939/55336038-56921380-54ad-11e9-8a67-be5ad8c90d93.png)